### DISCUSSION TITLE
Version 5 docs?

### DISCUSSION THREAD

**Discussion by daniellacoste** [storybookjs/storybook#24692] [Help] [answered]
Hey friends, I am new to SB and I am looking to learn more, but right now I'm on an old version (5.3.27). I hope we can migrate our SB to >=6 soon. I am looking for resources on how to use SB v5.3, but I can't find any official documentation about version 5 on the Storybook website (oldest docs are v6.0). Does anyone know where the legacy docs are stashed away? Thanks!

**Comment by jonniebigodes** [comment, marked as answer]
@daniellacoste thanks for reaching out to us with your question. We appreciate it 🙏 ! We still have it [here](https://storybook.js.org/docs/react/faq#is-it-possible-to-browse-the-documentation-for-past-versions-of-storybook). 

Let us know if it helped so that we can continue following up with you it or close this discussion.

Hope you have a great day.

Stay safe

**Reply by daniellacoste** [reply]
thanks for the quick reply, and thanks for the link!

**Reply by jonniebigodes** [reply]
No need to thank whatsoever, glad to help out.
